.circleci/config.yml

```diff
@@ -205,16 +205,14 @@ workflows:
         - ruby-2.7.2-rails-5.2
         filters:
           tags:
-            only: /^[0-9]+\.[0-9]+\.[0-9]+(\.?(RC|rc)[-\.]?\w*)?$/
-          branches:
-            ignore: /.*/
+            only: /^v?[0-9]+\.[0-9]+\.[0-9]+(\.?(RC|rc)[-\.]?\w*)?$/
```

Comment:
Running the `rake release` applies tags with a `v` prefix like `v4.1.0` which isn't picked up by the previous regex tag filter. See the [history of version tags](https://github.com/stitchfix/stitches/tags).

https://rubular.com/r/AVUeYARjiaIZnZ